contracts/voting/cwd-voting-cw721-staked/src/testing/execute.rs

```diff
@@ -0,0 +1,148 @@
+use cosmwasm_std::{Addr, Binary, Empty};
+use cw721::Cw721ExecuteMsg;
+use cw721_base::MintMsg;
+use cw_multi_test::{App, AppResponse, Executor};
+
+use anyhow::Result as AnyResult;
+use cw_utils::Duration;
+
+use crate::msg::ExecuteMsg;
+
+// Shorthand for an unchecked address.
+macro_rules! addr {
```

Comment:
cool!

Reply:
thanks! i've been reading this to learn about macros: https://veykril.github.io/tlborm/introduction.html